cpu/lm4f120/ldscripts/lm4f120.ld

```diff
@@ -0,0 +1,26 @@
+/*

```

Comment:
please rename this file to `lm4f120h5qr.ld`, so that if there are other CPUs added with different RAM/ROM configurations we are prepared for this...